bundles/sirix-core/src/main/java/org/sirix/api/ResourceManager.java

```diff
@@ -164,7 +164,7 @@ default R beginNodeReadOnlyTrx() {
    * @throws SirixThreadedException   if the thread is interrupted
    * @throws SirixUsageException      if the number of read-transactions is exceeded for a defined time
    */
-  R beginNodeReadOnlyTrx(@Nonnull Instant pointInTime);
+  R beginNodeReadOnlyTrx(final @Nonnull Instant pointInTime);
```

Comment:
I know that I sometimes specified final for interface method params. However, I'm not sure if that's even a good practice. Usually I don't do it anymore, as it has no effect, but maybe it's a kind of documentation at least. Even though I tend to remove them. What do you think and @JPDSousa?

Reply:
`final` just prevents parameter reassignment, which for interfaces has no effects. I think you can even "override" final in your implementation by simply not including it there. Therefore I think final only makes sense in methods which are implemented.

Also note that parameter reassignment is a bad practice [(source)](https://stackoverflow.com/questions/13507025/avoid-reassigning-parameters). `final` is just a way of making it explicit.

Reply:
you are right Joao, my bad. I will remove it.